### PR TITLE
add zh blog: Kubernetes Docs Updates, International Edition

### PR DESCRIPTION
**Pull request by liyuan198251**
Signed-off-by: 李元10098251 <li.yuan4@zte.com.cn>

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.13 Features: set Milestone to 1.13 and Base Branch to dev-1.13
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by liyuan198251** [comment]
@xiangpengzhao @tengqm thanks

File: content/zh/blog/_posts/2018-11-08-kubernetes-docs-update-i18n.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+layout: blog
+title: 'Kubernetes Docs Updates, International Edition'
+date: 2018-11-08
+---
+
+**Author**: Zach Corleissen (Linux Foundation)
+
+作为SIG Docs的联合主席，我很高兴地Kubernetes文档有一个完全成熟的本地化(l10n)工作流。
```
Comment (markthink) [inline content/zh/blog/_posts/2018-11-08-kubernetes-docs-update-i18n.md]: 原始的英文没有保留哦

Comment (markthink) [reply content/zh/blog/_posts/2018-11-08-kubernetes-docs-update-i18n.md]: 参考：https://github.com/k8smeetup/k8s-official-translation

**Comment by liyuan198251** [comment]
Got it.Thanks a lot.

**Comment by idealhack** [comment]
看起来工作已经转移到 #11082 

/close

**Pull request closed without merging**